### PR TITLE
updated tsinfer function, add cluster config

### PR DESCRIPTION
**Pull request by gmafrafortuna**
tsinfer function adding generate ancestors step
added parameter file for tsinfer function
added cluster configuration file -- so resources don't need to be set in the functions and it is easier to change

### CODE REVIEW

File: Snakemake/config/cluster.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    "mem" : 15000
+  },
+  "change_infoAA_vcf":
+  {
+    "time": "05:00:00",
+    "mem" : 20000
+  },
+  "phase":
+  {
+    "clusterSpec": "-P roslin_HighlanderLab -cwd -pe sharedmem 25"
```
Comment (janaobsteter) [inline Snakemake/config/cluster.json:50]: Why do we specify this again here? Does this apply just to the rules below this?

File: Snakemake/workflow/scripts/tsparam.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# parameters for tsinfer --- change accordingly
```
Comment (janaobsteter) [inline Snakemake/workflow/scripts/tsparam.py:1]: Will this one leave in the /scripts folder? And then the other python file will find it?

File: Snakemake/config/cluster.json (hunk 1 above)
Comment (gmafrafortuna) [reply Snakemake/config/cluster.json:50]: I'm not sure if drmaa pickup the CPU argument - but I know it works passing the -pe sharedmem so I put both until I fully figure out.

File: Snakemake/workflow/scripts/tsparam.py (hunk 2 above)
Comment (gmafrafortuna) [reply Snakemake/workflow/scripts/tsparam.py:1]: They have to be in the same folder (if they aren't that was my mistake)

**Pull request merged**